### ISSUE TITLE
Incorrect Signage on DWORD typedef

### ISSUE THREAD

**Issue by juntalis** [luapower/fs#1]
It looks like the [DWORD typedef](https://github.com/luapower/fs/blob/master/fs_win.lua#L24) in the Windows backend currently defines it as a signed 32-bit integer. This looks to conflict with the declaration from [Windef.h](https://github.com/wine-mirror/wine/blob/master/include/windef.h#L241) which defines it as an unsigned 32-bit integer.

*Or from the Windows 7 SDK*:

```c
typedef unsigned long       DWORD;
```

**Comment by capr** [comment]
Thanks for noticing. Fixed. How did you find the project so fast? I barely started working on it and I've not yet listed it on luapower.com.

**Comment by juntalis** [comment]
Haha - started following you on github a while back when you had (I believe) the inline dynasm extension under your user repositories. Saw this pop on my feed and thought it looked interesting.

Forked it intending to offer some symlink/hardlink code from a ffi-based lfs reimplementation I started but never finished a year ago, but it turns out I used the Vista+ API and never got around to the winioctl approach.
